“Run: 1”
/cm/local/apps/slurm/var/spool/job939682/slurm_script: line 23: $'\nconda create --name tpot2env -c conda-forge python=3.10\n': command not found
/cm/local/apps/slurm/var/spool/job939682/slurm_script: line 27: $'\npip install -r requirements.txt\n': command not found
RunStart
data/167104_True.pkl
  0%|          | 0/5 [00:00<?, ?it/s]Generation:   0%|          | 0/5 [00:00<?, ?it/s]Generation:  20%|██        | 1/5 [00:03<00:12,  3.16s/it]Generation:  40%|████      | 2/5 [00:07<00:11,  3.87s/it]Generation:  60%|██████    | 3/5 [00:10<00:07,  3.62s/it]Generation:  80%|████████  | 4/5 [00:13<00:03,  3.16s/it]Generation: 100%|██████████| 5/5 [00:15<00:00,  2.71s/it]Generation: 100%|██████████| 5/5 [00:15<00:00,  3.05s/it]
/home/hodesse/miniconda3/envs/tpot2env/lib/python3.10/site-packages/sklearn/preprocessing/_data.py:2785: UserWarning: n_quantiles (1194) is greater than the total number of samples (463). n_quantiles is set to n_samples.
  warnings.warn(
/home/hodesse/miniconda3/envs/tpot2env/lib/python3.10/site-packages/sklearn/preprocessing/_data.py:2785: UserWarning: n_quantiles (656) is greater than the total number of samples (463). n_quantiles is set to n_samples.
  warnings.warn(
Traceback (most recent call last):
  File "/common/hodesse/hpc_test/TPOT2_ensemble/hpc_test2.py", line 148, in <module>
    main()
  File "/common/hodesse/hpc_test/TPOT2_ensemble/hpc_test2.py", line 127, in main
    fitted_ensemble, individual_highest_accuracy = create_stacking_clf_gridsearch(est.pareto_front)
  File "/common/hodesse/hpc_test/TPOT2_ensemble/hpc_test2.py", line 75, in create_stacking_clf_gridsearch
    fitted_pipeline = pareto_front.iloc[i, 10].fit(X_train, y_train)
NameError: name 'X_train' is not defined
srun: error: esplhpc-cp001: task 0: Exited with exit code 1
